allocate.py

```diff
@@ -24,5 +24,6 @@
 allocator.read_entities()
 allocator.setup()
 allocator.allocate()
+allocator.assess()
```

Comment:
It's not clear to me why this is being put back, since we have a class to do analysis now.

Reply:
So the heuristic objects have an opportunity to report useful information (like which queues still have stuff in them).